### PR TITLE
change busybox to alpine:3.7

### PR DESCRIPTION
**Pull request by dvintel**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN for filename in $( \
     done
 
 
-FROM busybox:1.30.1
+FROM alpine:3.7
```
Comment (ajcasagrande) [inline Dockerfile:19]: I think this should be refactored to just base off the alpine from the beginning. Right now the builder is alpine and then `lib*.so` libs are copied over to busybox. If we want alpine as base image this is unnecessary.

**Review by ajcasagrande — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (dvintel) [reply Dockerfile]: Trying to keep the changes to a minimum, so we don't introduce more changes than required

Comment (dvintel) [reply Dockerfile:19]: Trying to keep changes to a minimum so we don't introduce any issues

Comment (ajcasagrande) [reply Dockerfile:19]: Approved

**Review by ajcasagrande — APPROVED** [review]

**Comment by dvintel** [comment]
reviewed with team during standup, this is a good mitigation for name resolution issues

**Pull request merged**